src/components/engagement/internship-position.resolver.ts

```diff
@@ -24,7 +20,7 @@ class InternshipPositionOptions {
   @Field(() => InternshipDomain, {
     nullable: true,
   })
-  domain: InternshipDomain | null;
+  domain: InternshipDomain | null | undefined;
```

Comment:
```suggestion
  domain: InternshipDomain | null;
```
Let's just make the use of this conform to null.
If it doesn't already we can just do `domain ?? null`